Procfile

```
@@ -0,0 +1,3 @@
+web: bundle exec puma -C config/puma.rb
+release: bundle exec rails db:migrate
+worker: bundle exec sidekiq -e production -q default
```

```suggestion
worker: bundle exec sidekiq -q default
```

No need to set the environment